e2e/features/case-progression.feature

```diff
@@ -504,4 +504,35 @@ Feature: Case progression
     And I should see the case details
     And I should see the hearing details
 
-    And I should not see the `Next step` field
\ No newline at end of file
+    And I should not see the `Next step` field
+
+    # start decision and reasons
+
+    When I switch to be a `Case Officer`
+    And I start decision and reasons
+    And I click the `Overview` tab
+
+    Then I should only see the `caseOfficer_decision` case progress image
+    And I should see the case details
+    And I should see the hearing details
+    And I should see the option `Send direction` for the `Next step` field
+    And I should see the option `Change the direction due date` for the `Next step` field
+    And I should see the option `Generate decision and reasons` for the `Next step` field
+
+    When I click the `Generate decision and reasons` link
+    Then I am on the `Generate decision and reasons` page
+    And I click the `Cancel` link
```

Comment:
Since 1544 is about sending the decision and reasons, I would expect us to add that here.  Please add that test case.  Please also put a comment stating the scenario as per previous tests.

Reply:
This is acceptable currently but will have to be revisited as tech debt.  The overview tab for 1544 is ahead of the actual feature and therefore can't be tested.  It is ok to release right now as it is impossible for any user to reach the state to see this.